### ISSUE TITLE
[0.12.0] Play 3 - NoClassDefFoundError: play/twirl/api/Template1

### ISSUE THREAD

**Issue by sentenza** [playframework/play-grpc#499]
After upgrading to Play 3.0.1 and Pekko I get the following error message at compile time if I exclude transitive dependencies with

```scala
excludeDependencies ++= Seq(
  // As of Play 3.0, groupId has changed to org.playframework; exclude transitive dependencies to the old artifacts
  ExclusionRule(organization = "com.typesafe.play")
)
```

NOTE: If I remove the previous exclusion rule I can compile the project but then I hit https://github.com/lloydmeta/enumeratum/issues/388

Play 3.0.1
Scala 2.13.12
JDK 17.0.9
play-grpc 0.12.0

```
[info] Compiling 1 protobuf files to /target/scala-2.13/pekko-grpc/main
--jvm_1_out: java.lang.reflect.InvocationTargetException
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  | => tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.pekko.grpc.sbt.GeneratorBridge$SandboxedProtocBridgeSbtPluginCodeGenerator.run(GeneratorBridge.scala:76)
	at protocbridge.frontend.PluginFrontend$.$anonfun$runWithBytes$1(PluginFrontend.scala:51)
	at scala.util.Try$.apply(Try.scala:213)
	at protocbridge.frontend.PluginFrontend$.runWithBytes(PluginFrontend.scala:51)
	at protocbridge.frontend.PluginFrontend$.runWithInputStream(PluginFrontend.scala:121)
	at protocbridge.frontend.PosixPluginFrontend$.$anonfun$prepare$2(PosixPluginFrontend.scala:40)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.impl.ExecutionContextImpl$DefaultThreadFactory$$anon$1$$anon$2.block(ExecutionContextImpl.scala:75)
	at java.base/java.util.concurrent.ForkJoinPool.compensatedBlock(ForkJoinPool.java:3451)
	at java.base/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
	at scala.concurrent.impl.ExecutionContextImpl$DefaultThreadFactory$$anon$1.blockOn(ExecutionContextImpl.scala:87)
	at scala.concurrent.package$.blocking(package.scala:146)
	at protocbridge.frontend.PosixPluginFrontend$.$anonfun$prepare$1(PosixPluginFrontend.scala:38)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NoClassDefFoundError: play/twirl/api/Template1
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:524)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:427)
	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:421)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:420)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:592)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaTraitCodeGenerator$.$anonfun$generateServiceFile$1(ScalaTraitCodeGenerator.scala:28)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.$anonfun$run$4(ScalaCodeGenerator.scala:80)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.$anonfun$run$4$adapted(ScalaCodeGenerator.scala:79)
	at scala.collection.immutable.Set$Set1.foreach(Set.scala:141)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.$anonfun$run$3(ScalaCodeGenerator.scala:79)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.$anonfun$run$3$adapted(ScalaCodeGenerator.scala:78)
	at scala.collection.Iterator.foreach(Iterator.scala:943)
	at scala.collection.Iterator.foreach$(Iterator.scala:943)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.run(ScalaCodeGenerator.scala:78)
	at org.apache.pekko.grpc.gen.CodeGenerator.run(CodeGenerator.scala:41)
	at org.apache.pekko.grpc.gen.CodeGenerator.run$(CodeGenerator.scala:38)
	at org.apache.pekko.grpc.gen.scaladsl.ScalaCodeGenerator.run(ScalaCodeGenerator.scala:27)
	... 31 more
Caused by: java.lang.ClassNotFoundException: play.twirl.api.Template1
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:592)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 57 more
	```

**Comment by ihostage** [comment]
I think it's related to Twirl version in Pekko gRPC ([project/plugins.sbt#L20](https://github.com/apache/incubator-pekko-grpc/blob/0b6851b519341cb9a9b3fdb8a7df9992f61b66c0/project/plugins.sbt#L20))

**Comment by pjfanning** [comment]
@sentenza The pekko-grpc snapshot used in https://github.com/pjfanning/play-grpc/tree/pekko-grpc-fix-test also has the sbt-twirl upgrade that @ihostage submitted. Could you try it?

**Comment by sentenza** [comment]
> @sentenza The pekko-grpc snapshot used in https://github.com/pjfanning/play-grpc/tree/pekko-grpc-fix-test also has the sbt-twirl upgrade that @ihostage submitted. Could you try it?

@pjfanning I've already tested the latest version of `play-grpc` (0.12.0). The issue was solved by upgrading `op-rabbit` to v4.0.0 that integrates the changes introduced in this PR https://github.com/pjfanning/op-rabbit/pull/6.
I consider this issue as already fixed, so I'm closing it. 

Thanks for the changes you suggested for `play-grpc` as well. :)
